x-pack/plugins/cloud/public/fullstory.ts

```diff
@@ -5,27 +5,24 @@
  * 2.0.
  */
 
-import { sha256 } from 'js-sha256';
```

Comment:
Note: we may want to move this into the async chunk that loads fullstory to reduce the initial page load.